### PR TITLE
Upgrade Chromium to version 111.0.5563.65

### PR DESCRIPTION
**Pull request by vladimir-ikryanov**
Please check the licenses.

### CODE REVIEW

**Comment by oleksandra-varavina** [comment]
The license file for freetype component references some additional files, could you please extract them for us and place in the component's folder together with the licenses?
https://github.com/TeamDev-IP/Chromium-Licences/blob/111.0.5563.64/chromium-licenses/third_party/freetype/src/LICENSE.TXT

Quote: 
> The FreeType License,  found in the file  **`docs/FTL.TXT`**, which is
     similar to the  original BSD license...

> The contributed  BDF and PCF  drivers come  with a license  similar to
> that  of the  X Window  System.   It is  compatible to  the above  two
> licenses (see files **`src/bdf/README`**  and **`src/pcf/README`**) ...

File: chromium-licenses/third_party/mozilla/LICENSE
Comment (oleksandra-varavina) [inline chromium-licenses/third_party/mozilla/LICENSE:1]: Could you please make one more check? We've seen this file in and out already. Could you please check that mozilla component is really not used by the build?

Comment (vladimir-ikryanov) [reply chromium-licenses/third_party/mozilla/LICENSE:1]: @oleksandra-varavina this file is marked as deleted in Chromium 111.

**Comment by vladimir-ikryanov** [comment]
@oleksandra-varavina could you please use the INLINE comments instead of trying to explain about which file you are talking about in the comments? It's much easier to discuss the text of a file in the **Files changed** tab.

File: chromium-licenses/third_party/freetype/src/LICENSE.TXT [hunk 1]
```diff
@@ -29,12 +29,16 @@ The contributed  BDF and PCF  drivers come  with a license  similar to
...
 licenses (see files `src/bdf/README`  and `src/pcf/README`).  The same
 holds   for   the   source    code   files   `src/base/fthash.c`   and
-`include/freetype/internal/fthash.h`; they wer part  of the BDF driver
+`include/freetype/internal/fthash.h`; they were part of the BDF driver
 in earlier FreeType versions.
 
 The gzip  module uses the  zlib license (see  `src/gzip/zlib.h`) which
 too is compatible to the above two licenses.
 
+The files `src/autofit/ft-hb.c` and `src/autofit/ft-hb.h` contain code
```
Comment (vladimir-ikryanov) [inline chromium-licenses/third_party/freetype/src/LICENSE.TXT:38]: @oleksandra-varavina JFYI: this is an example of the inline comment in GitHub PR.

Comment (oleksandra-varavina) [reply chromium-licenses/third_party/freetype/src/LICENSE.TXT:38]: Thanks for showing me this. Will use it.

**Review by oleksandra-varavina — APPROVED** [review]
All reviewed, no red flag issues.

**Pull request merged**